resources/provider/provider.go

```diff
@@ -151,6 +151,7 @@ func Provider() *provider.Provider {
 			"efs.filesystems":                       efs.EfsFilesystems(),
 			"eks.clusters":                          eks.EksClusters(),
 			"elasticbeanstalk.applications":         elasticbeanstalk.ElasticbeanstalkApplications(),
+			"elasticbeanstalk.application_versions": elasticbeanstalk.ApplicationVersions(),
```

Comment:
The name without the prefix `elasticbeanstalk.ApplicationVersions` vs `elasticbeanstalk.ElasticbeanstalkApplicationVersions` was generated by `cq-gen`, so I kept it this way.

Happy to rename it